pkg/webview/view.proto

```diff
@@ -8,20 +8,31 @@ import "pkg/webview/log.proto";
 
 option go_package = "github.com/windmilleng/tilt/pkg/webview";
 
+enum BuildType {
+  IMAGE = 0;
```

Comment:
in protobufs, the 0 enum value is special, and is generally used for some sort of natural zero value or unspecified value. See:
https://developers.google.com/protocol-buffers/docs/style#enums

because it indicates the case where the field wasn't populated. I would not expect IMAGE to be a zero value

Reply:
good tip, ty!